src/components/TopBar/ExportMenu.tsx

```diff
@@ -0,0 +1,380 @@
+<rect x="18.4658" y="12.4966" width="20.0083" height="30.0125" fill="#543CF0"/>
+</svg`;
+
+const toDataUrl = (img: unknown): string | undefined => {
+  if (!img) return undefined;
+  if (typeof img === 'string') return img;
+  if (typeof img === 'object' && img !== null && 'data' in (img as Record<string, unknown>)) {
+    const d = (img as Record<string, unknown>).data;
+    return typeof d === 'string' ? d : undefined;
+  }
+  return undefined;
+};
+
+type ExportMenuProps = {onDone?: () => void};
+
+export default function ExportMenu({onDone}: ExportMenuProps): JSX.Element {
+  const {t} = useTranslation();
+  const {formatNumber} = NumberFormatter(i18n.language, 1, 0);
+  const {t: tBackend, i18n: i18nBackend} = useTranslation('backend');
+  const {t: tGlobal} = useTranslation('global');
+  const {get} = useExportingRegistry();
+  const {compartments, referenceDateValues, scenarioCardData} = useContext(DataContext)!;
+
+  const selectedScenario = useAppSelector((state) => state.dataSelection.scenario);
+  const scenariosState = useAppSelector((state) => state.dataSelection.scenarios);
+  const selectedDistrict = useAppSelector((state) => state.dataSelection.district);
+  const selectedDate = useAppSelector((state) => state.dataSelection.date);
+  const referenceDay = useAppSelector((state) => state.dataSelection.simulationStart);
+
+  const languageSuffix = i18nBackend.language === 'de' ? '-de' : '-en';
```

Comment:
We should keep this flexible just in case other languages would be added at some point.
Generally I would just use the `i18nBackend.language` property directly as suffix, the only issue we could run into is if the browser detects a different language and we haven't set the new language yet (i.e. via language picker).
```suggestion
  const languageSuffix = `-${i18nBackend.language}`;
```